client/apps/plugin-status/health.js

```diff
@@ -25,8 +27,32 @@ const HealthView = ( { translate, healthItems } ) => {
 			<Indicator
 				title={translate('Jetpack')}
 				state={healthItems.jetpack.state}
-				message={healthItems.jetpack.message}/>
-			<WooCommerceServicesIndicator />
+				message={healthItems.jetpack.message}
+			/>
+			<Indicator
+				title={translate('Automatic Taxes')}
```

Comment:
The taxes feature is referred to as "Automated" rather than "Automatic" throughout the settings and docs. Let's change all occurrences of "Automatic" to "Automated" for consistency.